pkg/idtools/idtools_windows.go

```diff
@@ -8,7 +8,7 @@ import (
 
 // Platforms such as Windows do not support the UID/GID concept. So make this
 // just a wrapper around system.MkdirAll.
-func mkdirAs(path string, mode os.FileMode, ownerUID, ownerGID int, mkAll, chownExisting bool) error {
+func mkdirAs(path string, mode os.FileMode, identity Identity, mkAll, chownExisting bool) error {
```

Comment:
Now that `Identity` also has a `SID` maybe expand the comment to explain why it can't be used here to change the ownership?

Reply:
I'll update the comment. This can be used to set the security descriptor of the target, however, this is more inline with 'chmod' as opposed to 'chown'. chown requires special privileges in order to set an owner with a SID not present/enabled on the caller's token.